helm/nessie/templates/servicemonitor.yaml

```diff
@@ -20,7 +20,8 @@
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
-  name: {{ template "nessie.fullname" . }}
+  name: {{ include "nessie.fullname" . }}
```

Comment:
Intentional change?

Reply:
Yes, to align with other occurrences of this template inclusion. (In this context, using `template` or `include` yields the same result.)